### PR TITLE
Add python pexpect

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
[test rpm]

File: comps/comps-foreman-plugins-el8.xml [hunk 1]
```diff
@@ -22,6 +22,7 @@
       <packagereq type="default">nodejs-react-json-tree</packagereq>
       <packagereq type="default">puppet-foreman_scap_client</packagereq>
       <packagereq type="default">puppetlabs-stdlib</packagereq>
+      <packagereq type="default">python38-pexpect</packagereq>
```
Comment (evgeni) [inline comps/comps-foreman-plugins-el8.xml:25]:
```suggestion
      <packagereq type="default">python39-pexpect</packagereq>
```

**Comment by evgeni** [comment]
the "add mock" commit is in many of your PRs, and it's always using 3.8 and not 3.9.

**Comment by ehelms** [comment]
> the "add mock" commit is in many of your PRs, and it's always using 3.8 and not 3.9.

Broke it out to here https://github.com/theforeman/foreman-packaging/pull/8500

**Pull request merged**